web-frontend/src/pages/Assets/Assets.tsx

```diff
@@ -1,15 +1,108 @@
-import Placeholder from '@/layouts/Placeholder/Placeholder';
 import { WalletLayout, MenuItem } from '@/layouts/WalletLayout/WalletLayout';
 import Intl from '@/i18n/i18n';
-import { FiDisc } from 'react-icons/fi';
+import { FiPlus } from 'react-icons/fi';
+import {
+  Button,
+  Input,
+  MassaLogo,
+  Mrc20,
+  Mrc721,
+  PopupModal,
+  PopupModalContent,
+  PopupModalHeader,
+  Token,
+} from '@massalabs/react-ui-kit';
+import { useState } from 'react';
+
+// TODO : Connect to /api/assets
 
 function Assets() {
+  const [modal, setModal] = useState(false);
+
+  // mock token list data
+  const massaToken = {
+    logo: <MassaLogo size={40} />,
+    formattedBalance: '000,000,000.00',
+    name: 'Massa',
+    symbol: 'SYMBL',
+    rawBalance: '000,000,000.000000000',
+  };
+
+  const mrc20Token = {
+    logo: <Mrc20 size={40} />,
+    formattedBalance: '000,000,000.00',
+    name: 'Mrc20',
+    symbol: 'SYMBL',
+    rawBalance: '000,000,000.000000000',
+  };
+
+  const mrc721Token = {
+    logo: <Mrc721 size={40} />,
+    formattedBalance: '000,000,000.00',
+    name: 'Mrc721',
+    symbol: 'SYMBL',
+    rawBalance: '000,000,000.000000000',
+  };
+
+  const tokenList = [massaToken, mrc20Token, mrc721Token];
+
   return (
     <WalletLayout menuItem={MenuItem.Assets}>
-      <Placeholder
-        message={Intl.t('placeholder.teaser-assets-page')}
-        icon={<FiDisc size={114} />}
-      />
+      <div className="flex flex-col bg-secondary p-10 h-fit w-2/3 rounded-lg">
+        <div className="flex flex-row items-center justify-between w-full mb-6">
+          <div> {Intl.t('assets.title')}</div>
+          <Button
+            customClass="w-fit"
+            preIcon={<FiPlus size={114} />}
+            onClick={() => {
+              setModal(true);
+            }}
+          >
+            {Intl.t('assets.import')}
+          </Button>
+        </div>
+        <div className="flex flex-col w-full h-fit bg-primary rounded-lg gap-4 p-8">
+          {tokenList.map((token, index) => (
+            <Token
+              token={token}
+              key={index}
+              // This will change when we have token consts
+              disable={token?.name === 'Massa' ? true : false}
```

Comment:
Why you need to disabled `Token` component when the token name is Massa ? 🤔

Reply:
because the user shouldn't be able to delete the massa token